batch/Makefile

```diff
@@ -23,7 +23,7 @@ build:
 	docker build -t batch -f Dockerfile.out --cache-from batch,$(BATCH_LATEST),service-base ..
 	-docker pull python:3.7-slim-stretch
 	-docker pull $(BATCH_WORKER_LATEST)
-	docker build -t batch-worker -f Dockerfile.worker --cache-from batch-worker,$(BATCH_WORKER_LATEST),python:3.7-slim-stretch ..
+	docker build -t batch-worker -f Dockerfile.worker.out --cache-from batch-worker,$(BATCH_WORKER_LATEST),python:3.7-slim-stretch ..
```

Comment:
I think this should be Dockerfile.worker

Reply:
good catch